chia/rpc/wallet_rpc_api.py

```diff
@@ -2978,13 +2980,16 @@ async def get_farmed_amount(self, request) -> EndpointResult:
             # .get_farming_rewards() above queries for only confirmed records.  This
             # could be hinted by making TransactionRecord generic but streamable can't
             # handle that presently.  Existing code would have raised an exception
-            # anyways if this were to fail and we already have an assert below.
+            # anyway if this were to fail and we already have an assert below.
             assert height is not None
             if record.type == TransactionType.FEE_REWARD:
-                fee_amount += record.amount - calculate_base_farmer_reward(height)
-                farmer_reward_amount += calculate_base_farmer_reward(height)
+                base_farmer_reward = calculate_base_farmer_reward(height)
+                fee_amount += record.amount - base_farmer_reward
+                farmer_reward_amount += base_farmer_reward
+                blocks_won += 1
             if height > last_height_farmed:
                 last_height_farmed = height
+                last_time_farmed = await self.service.get_timestamp_for_height(height)
```

Comment:
we could save even more time by moving this call out of the loop too, and just call it once in `last_height_farmed`

Reply:
Thanks, this is really important comment.
@felixbrucker I've fixed the issue adderssed by this comment.
Performance degradation should be resolved after this PR is merged.